src/pretix/base/models/discount.py

```diff
@@ -171,6 +173,17 @@ class Discount(LoggedModel):
                     "access to sold-out quota will still receive the discount."),
     )
 
+    subevent_date_from = models.DateTimeField(
+        verbose_name=_("Available for dates from"),
+        null=True,
+        blank=True,
+    )
+    subevent_date_until = models.DateTimeField(
+        verbose_name=_("Available for dates until"),
```

Comment:
```suggestion
        verbose_name=pgettext_lazy("subevent", "Available for dates starting until"),
```

Reply:
```suggestion
        verbose_name=pgettext_lazy("subevent", "Available for dates starting before"),
```
sounds better then, right?